examples/java/pom.xml

```diff
@@ -273,7 +273,6 @@
       <artifactId>beam-runners-direct-java</artifactId>
       <version>${project.version}</version>
       <scope>runtime</scope>
-      <optional>true</optional>
```

Comment:
Please remove from this PR if it is not necessary for the PR. Bundling cleanup and functional changes is generally not a good idea -- it complicates review, testing, identification of issues, and rollback of bad changes.